fili-core/src/main/java/com/yahoo/bard/webservice/table/resolver/GranularityComparator.java

```diff
@@ -17,6 +17,15 @@
     private static final Logger LOG = LoggerFactory.getLogger(GranularityComparator.class);
 
     /**
+     * Factory method for creating new GranularityComparator instance.
+     *
+     * @return a new GranularityComparator instance
+     */
+    public static GranularityComparator getInstance() {
+        return new GranularityComparator();
```

Comment:
This class should create an instance statically, cache it, and then this method can return the cached instance every time. We explicitly _don't_ want to be making new instances all the time because there's no point (ie. no difference between the instances)